go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/StudioSol/async
 
 go 1.13
+
+require (
+	github.com/smartystreets/goconvey v1.6.4
+	github.com/stretchr/testify v1.6.1
```

Comment:
Can you remove these dependencies? `go mod tidy`
It is only used in tests. It helps to reduce dependencies in `go get` step for end-users.

Reply:
yes, i have removed